### PR TITLE
Modify README files and add a guidance

### PR DESCRIPTION
**Pull request by tsuting**
1. Modify README for deepstream
2. Modify README for this whole repo
3. Add setup guidance for AGX Xavier
4. Add requirements.txt
5. Fix typo in the code

### CODE REVIEW

File: py-deepstream-text-detection/README.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
-# Introduction 
-OCR with the paddlepaddle detection model and object tracking on deepStream. Input is from an usb camera.
```
Comment (xiaolul) [inline py-deepstream-text-detection/README.md:2]: an USB camera -> a USB camera

File: py-deepstream-text-detection/README.md [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
 * Add performance measurement (fps)
 
-# Prerequisite
+Most of the code are copied from [`deepstream_python_apps/apps/`](https://github.com/NVIDIA-AI-IOT/deepstream_python_apps/tree/master/apps). The code for PaddlePaddle decoder is from [here](https://github.com/PaddlePaddle/PaddleOCR/blob/63cd23ab1c3f092666aad627a27dab1588aca83f/ppocr/postprocess/__init__.py) and [here](https://github.com/PaddlePaddle/PaddleOCR/blob/63cd23ab1c3f092666aad627a27dab1588aca83f/ppocr/postprocess/db_postprocess.py
+).
+
+## Prerequisite
 
-  * Remember to add this folder under `deepstream_python_apps`
-  * Place the onnx model under the same layer as the python files
```
Comment (xiaolul) [inline py-deepstream-text-detection/README.md:14]: same layer -> same directory (or same folder)

File: py-deepstream-text-detection/README.md [hunk 3]
```diff
@@ -1,23 +1,31 @@
...
-  * If you would like to run DeepSORT object tracker, please follow the documentation to prepare the model.
-    * /opt/nvidia/deepstream/deepstream/sources/tracker_DeepSORT/README
+* A webcam
```
Comment (xiaolul) [inline py-deepstream-text-detection/README.md]: A USB webcam

File: py-deepstream-text-detection/README.md [hunk 4]
```diff
@@ -1,23 +1,31 @@
...
+* Follow [the guidance](setup_guidance_AGX_Xavier.md) to setup your device and environment if you're using Jetson AGX Xavier Developer Kit.
+* Remember to add this folder under `deepstream_python_apps`.
+* Place the onnx model under the same layer as the python files or please check the `onnx-file` in `ocr_pgie_config.txt`.
```
Comment (xiaolul) [inline py-deepstream-text-detection/README.md]: same layer -> same directory (or same folder)

File: py-deepstream-text-detection/README.md [hunk 5]
```diff
@@ -41,10 +49,32 @@ optional arguments:
                         The approach to do objec tracking, default=NvDCF
 ```
 
-# Run
+## Prepare environments
+
+`pip3 install -r requirements.txt`
```
Comment (xiaolul) [inline py-deepstream-text-detection/README.md]: do you mean requirements from the parent folder?

File: py-deepstream-text-detection/setup_guidance_AGX_Xavier.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+* Cudnn 8.2
+
+Furthermore, we will build and install `Python bindings of deepStream SDK` on the device.
+
+**This guidance might not be 100% correct and there might be other ways to do so (like Docker).**
+
+## Prerequisite
+
+* A machine with Ubuntu Linux x64 Version 18.04 or 16.04
+* An USB keyboard and a mouse
```
Comment (xiaolul) [inline py-deepstream-text-detection/setup_guidance_AGX_Xavier.md]: An USB -> a USB

**Review by xiaolul — APPROVED** [review]

File: py-deepstream-text-detection/README.md (hunk 5 above)
Comment (tsuting) [reply py-deepstream-text-detection/README.md]: Yes, I changed to `../requirements.txt`

**Pull request merged**